apps/meteor/app/utils/lib/mimeTypes.ts

```diff
@@ -4,6 +4,8 @@ mime.types.wav = 'audio/wav';
 mime.types.lst = 'text/plain';
 mime.define('image/vnd.microsoft.icon', { source: '', extensions: ['ico'] }, mime.dupAppend);
 mime.define('image/x-icon', { source: '', extensions: ['ico'] }, mime.dupAppend);
+mime.define('audio/aac', { source: '', extensions: ['aac'] }, mime.dupAppend);
+mime.types.aac = 'audio/aac';
```

Comment:
```suggestion
mime.define('audio/aac', { source: '', extensions: ['aac'] }, mime.dupOverwrite);
```
This works just the same (the assignment in the second line works like an overwrite, which invalidates the first line)

Reply:
Good one, done